pypika/tests/test_criterions.py

```diff
@@ -333,6 +332,45 @@ def test__function_isin(self):
         self.assertEqual('COALESCE("isin"."foo",0) IN (0,1)', str(c2))
 
 
+class IsNotInTests(unittest.TestCase):
+    t = Table('abc', alias='isnotin')
```

Comment:
Could you please change the alias to something like `test_table`?  It's not immediately clear in the test assertions and gave me the impression at first of something not being serialized correctly.